data/test/units/test_attack_dummy.cfg

```diff
@@ -0,0 +1,30 @@
+#textdomain wesnoth-test
+
+
+# A Unit with simple stats for testing attack & ability releated functions..
+
+[unit_type]
+    id="test_attack_dummy"
+    name=_ "dummy_unit^Test Unit Attack"
+    image="misc/blank-hex.png"
```

Comment:
I suggest using something other than blank, in case someone wants to debug a test using this with `--showgui`. Perhaps something from the _scenery_ directory?

Reply:
probably

Reply:
I've taken a copy of this in #7738, renamed it "Melee Test Quintain", and used a sword as the `image`.